.test-infra/jenkins/job_PostCommit_Java_Nexmark_Dataflow.groovy

```diff
@@ -31,163 +31,12 @@ import static NexmarkDatabaseProperties.nexmarkInfluxDBArgs
-              ' -Pnexmark.args="' +
-              [
-                commonJobProperties.mapToArgString(nexmarkBigQueryArgs),
-                commonJobProperties.mapToArgString(nexmarkInfluxDBArgs),
-                '--runner=DataflowRunner',
-                '--region=us-central1',
-                '--numWorkers=4',
-                '--maxNumWorkers=4',
-                '--autoscalingAlgorithm=NONE',
-                '--nexmarkParallel=16',
-                '--queryLanguage=sql',
-                '--streaming=true',
-                '--suite=STRESS',
-                '--manageResources=false',
-                '--monitorJobs=true',
-                '--enforceEncodability=true',
-                '--enforceImmutability=true"'
-              ].join(' '))
-        }
-        shell('echo "*** RUN NEXMARK IN ZETASQL BATCH MODE USING DATAFLOW RUNNER ***"')
-        gradle {
-          rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':sdks:java:testing:nexmark:run')
-          commonJobProperties.setGradleSwitches(delegate)
-          switches('-Pnexmark.runner=":runners:google-cloud-dataflow-java"' +
-              ' -Pnexmark.args="' +
-              [
-                commonJobProperties.mapToArgString(nexmarkBigQueryArgs),
-                commonJobProperties.mapToArgString(nexmarkInfluxDBArgs),
-                '--runner=DataflowRunner',
-                '--region=us-central1',
-                '--numWorkers=4',
-                '--maxNumWorkers=4',
-                '--autoscalingAlgorithm=NONE',
-                '--nexmarkParallel=16',
-                '--queryLanguage=zetasql',
-                '--streaming=false',
-                '--suite=STRESS',
-                '--manageResources=false',
-                '--monitorJobs=true',
-                '--enforceEncodability=true',
-                '--enforceImmutability=true"'
-              ].join(' '))
-        }
-        shell('echo "*** RUN NEXMARK IN ZETASQL STREAMING MODE USING DATAFLOW RUNNER ***"')
-        gradle {
-          rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':sdks:java:testing:nexmark:run')
-          commonJobProperties.setGradleSwitches(delegate)
-          switches('-Pnexmark.runner=":runners:google-cloud-dataflow-java"' +
-              ' -Pnexmark.args="' +
-              [
-                commonJobProperties.mapToArgString(nexmarkBigQueryArgs),
-                commonJobProperties.mapToArgString(nexmarkInfluxDBArgs),
-                '--runner=DataflowRunner',
-                '--region=us-central1',
-                '--numWorkers=4',
-                '--maxNumWorkers=4',
-                '--autoscalingAlgorithm=NONE',
-                '--nexmarkParallel=16',
-                '--queryLanguage=zetasql',
-                '--streaming=true',
-                '--suite=STRESS',
-                '--manageResources=false',
-                '--monitorJobs=true',
```

Comment:
Do we need these options?

Reply:
Set here: https://github.com/apache/beam/blob/c2d4e5162afdc5ad9d32da7eca681581322e515f/.test-infra/jenkins/NexmarkBuilder.groovy#L34